src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -532,3 +532,10 @@ def copy_from_household_member(
             )
 
     return pop
+
+
+def write_metadata_file(final_output_dir: Path, label_version: str) -> None:
+    data_version = pd.Series(label_version)
```

Comment:
NOt sure what the ultimate goal of this metadata file is, but might be worth adding all the things (like all args used as input)

Reply:
We haven't fleshed out everything that will need to be in this file. THe main purpose at some point is going to be to calculate different proportions of the datasets to use for noising in pseudopeople and not have to calculate these values in pseudopeople and do it in post-processing. I'm not sure how helpful run params would be but I'll think about it because adding more stuff now wouldn't hurt.

Reply:
This is just to create the metadata file and validate the data version using it for now. We will almost certainly add more things to it.

Reply:
I don't remember everything we've talked about putting in the metadata file, but it seems like it would be good to think through it and come up with a list of ideas. I think a lot of the use cases came up in the context of providing useful user warnings without having to read the whole dataset. Here are the ones I remember:

- Available years for each dataset
- Available states for each dataset
- What locale the data is for (Anytown, RI, or USA)
- Version number of the data
- Fraction of eligible rows for various noise types (nicknames, copy from household member, guardian-based duplication) -- this may be less relevant if @zmbc [implements concatenating the data before adding noise](https://ihme.slack.com/archives/C02KUQ9LX32/p1698265517189139) rather than noising each shard separately, though we'd still need it for noising in parallel, e.g., using Modin
- I think there was some other issue with nicknames that we talked about potentially putting in a metadata file, but it would require storing a lot of data. I forget exactly what it was -- does anyone remember?

Can anyone think of other things that should go in the metadata file?